clang/include/clang/3C/ProgramInfo.h

```diff
@@ -45,7 +45,16 @@ typedef std::pair<CVarSet, BKeySet> CSetBkeyPair;
 
 class ProgramInfo : public ProgramVariableAdder {
 public:
-  typedef std::map<unsigned int, ConstraintVariable *> CallTypeParamBindingsT;
+
+  // This map holds similar information as the type variable map in
+  // ConstraintBuilder.cpp, but it is stored in a form that is usable during
+  // rewriting.
+  // The pair of CVs are the type param constraint (first) and an optional
+  // constraint for the single identifier argument, used to get the generic
+  // index if the first is void.
+  typedef std::map<unsigned int,
```

Comment:
Can this pair be moved into a struct so that the fields can be accessed by name instead of `first` and `second`?

Reply:
Yes, done.